students/TinaB/lesson07/test_html_render.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+import html_render as hr
+from io import StringIO
+import unittest
+
+
+class Test_Html_Render(unittest.TestCase):
+
+    def setUp(self):
+        self.html = hr.Html()
+        self.head = hr.Head(hr.Title('The Title is Title'))
+        self.body = hr.Body()
+        self.paragraph = hr.P('This is my paragraph', style="color:blue;")
+        self.horizontal_rule = hr.Hr()
+        self.anchor = hr.A('http://chicktech.org', 'link')
+        self.unorderedlist = hr.Ul(id="UL_List", style="line-height:100%")
+        self.headertwo = hr.H(3, 'This is a Header 3 ')
+        self.meta = hr.Meta(charset="UTF-8")
```

Comment:
it probably makes sense to instantiate as part of each new test and not in setup here because you aren't actualy using this in each test.